doc/nrf/known_issues.rst

```diff
@@ -659,6 +659,14 @@ Subsystems
 Bluetooth® LE
 =============
 
+.. rst-class:: v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-1 v1-5-0
+
+DRGN-16506: REM does not stop the HF Clock in between events when 'no prepare' is configured
+  When an MPSL timeslot is requested with MPSL_TIMESLOT_HFCLK_CFG_NO_GUARANTEE, the HF clock will be kept running between timeslot events.
+  This results in increased of power consumption in idle.
+
+  **Workaround:** Use MPSL_TIMESLOT_HFCLK_CFG_XTAL_GUARANTEED instead MPSL_TIMESLOT_HFCLK_CFG_NO_GUARANTEE to requested an timeslot.
```

Comment:
```suggestion
  **Workaround:** Use MPSL_TIMESLOT_HFCLK_CFG_XTAL_GUARANTEED instead of MPSL_TIMESLOT_HFCLK_CFG_NO_GUARANTEE to request a timeslot.
```